src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -461,6 +486,14 @@ public FormValidation doCheckServer(@QueryParameter String value, @QueryParamete
             }
         }
 
+        @POST
+        public FormValidation doCheckManagerPasswordSecret(@QueryParameter String managerPasswordSecret) {
```

Comment:
Always do a permissions check
```suggestion
        public FormValidation doCheckManagerPasswordSecret(@QueryParameter String managerPasswordSecret) {
            if(!Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
                return FormValidation.ok();
            }
```

Reply:
Taken care of the suggestion 👍